components/Menu.tsx

```diff
@@ -138,7 +138,57 @@ export default function Menu({ open, setSidebarOpen }: MenuProps) {
                 <MenuLink href="/intro">Intro</MenuLink>
                 <MenuLink href="/legal">Legal</MenuLink>
                 <MenuLink href="/policy">Policy</MenuLink>
-                <MenuLink href="/reference">Reference</MenuLink>
+                <MenuLink 
+                  href="/reference"
+                  subLinks={
+                    <Fragment>
+                      <MenuSubLink href="/reference#api-versioning">
+                        API Versioning
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#error-messages">
+                        Error Messages
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#authentication">
+                        Authentication
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#encryption">
+                        Encryption
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#snowflakes">
+                        Snowflakes
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#id-serialization">
+                        ID Serialization
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#iso8601-date/time">
```

Comment:
```suggestion
                      <MenuSubLink href="/reference#iso8601-datetime">
```
I think the / gets stripped?